models/vehicle.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type Vehicle struct {
+	Id    int64  `json:"id"`
```

Comment:
In Go, all acronyms should be written in ALL CAPS.

```suggestion
	ID    int64  `json:"id"`
```